web/src/main/java/com/microsoft/applicationinsights/web/extensibility/modules/WebRequestTrackingTelemetryModule.java

```diff
@@ -59,17 +57,31 @@
      */
     private final String W3C_BACKCOMPAT_PARAMETER = "enableW3CBackCompat";
 
+    /**
+     * The {@link RequestTelemetryContext} instance propogated from
+     * {@link com.microsoft.applicationinsights.web.internal.httputils.HttpServerHandler}
+     */
+    private RequestTelemetryContext requestTelemetryContext;
+
+    public void setRequestTelemetryContext(
+        RequestTelemetryContext requestTelemetryContext) {
+        this.requestTelemetryContext = requestTelemetryContext;
+    }
+
+    /** Used for test */
```

Comment:
com.google.common.annotations.VisibleForTesting?

Reply:
We removed all the dependency to Guava. This is Guava annotation.